App.test.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 import { render } from 'react-native-testing-library';
+import { shallow } from 'enzyme';
```

Comment:
So finally we have enzyme :smile: :+1:

Reply:
Yes, looks like it's still useful for shallow rendering :smile: